tools/add_module.py

```diff
@@ -204,4 +205,7 @@ def main(argv=None):
 
 
 if __name__ == "__main__":
+  if os.getenv("BUILD_WORKSPACE_DIRECTORY"):
```

Comment:
nit: I'd suggest a comment here like "Under 'bazel run' we want to run within the source folder instead of the execroot"